depthai_common_msgs/include/depthai_common_msgs/plugin/CameraBoardSocket.h

```diff
@@ -0,0 +1,6 @@
+#include <depthai-shared/datatype/CameraBoardSocket.hpp>
+
+//#define DEPTHAI_COMMON_MSGS_MESSAGE_CAMERABOARDSOCKET_PLUGIN_CLASS_BODY \
+//    MSGPACK_DEFINE()
```

Comment:
Any reason to keep the commented code?

Reply:
I just kept these to show these are enums. Enum should be handled properly later. I will delete these.